worker/raft/manifold_test.go

```diff
@@ -142,6 +143,17 @@ func (s *ManifoldSuite) TestOutput(c *gc.C) {
 	s.worker.CheckCallNames(c, "Raft")
 }
 
+func (s *ManifoldSuite) TestLogStoreOutput(c *gc.C) {
+	w := s.startWorkerClean(c)
```

Comment:
`ManifoldSuite.newWorker` (which is passed into the manifold) returns a `*mockRaftWorker` rather than a real running goroutine, so there's not really a need to stop it.

Reply:
Ok